src/literals.jl

```diff
@@ -0,0 +1,16 @@
+# "Fundamental" literals
+crawl(ex::String) = JSSymbol(js"$ex")
+crawl(b::Bool) = crawl(Symbol(b))
+
+# Symbols
+# We sub-dispatch on symbols to allow for special handling of certain keywords.
+function crawl(sym::Symbol)
+    if hasmethod(crawl, (Val{:symbol}, Val{sym}))
+        return crawl(Val(:symbol), Val(sym))
+    end
+    return JSSymbol(sym)
```

Comment:
This also could be a fallback `crawl(::Val{:symbol}, ::Val{sym}) where {sym} = JSSymbol(sym)`